Add metadata for fdroid build (#2373)
jamulussoftware/jamulus: metadata/en-US/full_description.txt

> This is a listening client for Jamulus. Please note that you need to use a <b>cellular or ethernet connection</b> in order to use this app. WiFi will lead to massively degraded audio quality. Also note that not all devices provide low enough audio latency for Jamming, which is why you should use this App as listening client.

Author: Is this too much pushing people back?

Reviewer: tl;dr version: "Jamming: YMMV".

Yes, I think it's pushing too much.  I'd say maybe that optimisation on Android has not yet been addressed and that experiences over various networking approaches may vary.  (Is WiFi bad for just listening?  Really?)

I've not actually attempted _use_ at all with the Android version, only verified an early version could connect and "appear to be working".

Reviewer: I also notice Google had already found these translations when I went to translate the German to English and gave me this exact text..!

Author: Interesting. I translated English to German by hand ;-). 

Ok. So what would you suggest?

```suggestion
This is a listening client for Jamulus. Please note that you should to use a <b>cellular or ethernet connection</b>, since WiFi will lead to degraded audio quality. Also note that latency optimization on Android has not been addressed and experiences will vary from device to device. Therefore this should be seen as listening client.
```

Reviewer 2: > Maybe
> 
> This is a listening client for Jamulus. Because latency optimization on Android has not been addressed, experiences will vary from device to device, so trying to participate in jams isn't recommended. For best performance, listen only over an ethernet or (4G?)/5G cellular connection (as WiFi, even for listening, isn't suitable).
> 
> * Is 4G suitable?
> * Is WiFi _not_ suitable just for listening?

I would have thought with auto-set jitter buffer, that both a good 4G signal and Wifi are fine for just listening. I have listened over Wifi with no problems, and have also participated in Zoom calls with the remote person on 4G in good quality, so would expect adequate performance for Jamulus, if latency is not an issue.

Reviewer: [802.11ac](https://en.wikipedia.org/wiki/Wi-Fi#Versions_and_generations) or ax WiFi, I guess, rather than earlier.

Author: On iOS 4G is perfectly suitable, On Android I think it’s a bit too much latency for untrained musicians if they want to play along (I didn’t try on a modern device).

Reviewer: OK, based on the above, I'd make it _very_ undirective.

Welcome to Jamulus. Latency optimization on Android has not been addressed and experiences will vary from device to device, so trying to participate in jams should only be attempted when you're sure you have a good connection.  The usual recommendations apply - a stable connection is essential (ethernet, 4G/5G cellular, preferably not WiFi).  Try listening only first and see if the latency is suitable before trying to join in and jam.

Author: What about 

```suggestion
Welcome to Jamulus. please note, latency optimization on Android has not been addressed and experiences will vary strongly from device to device. Trying to participate in jams should only be attempted when you're sure you have a good connection. The usual recommendations apply - a stable connection (4G/5G cellular; Ethernet. Stable, good quality WiFi signal might work, but is discouraged) and wired, non-Bluetooth headphones are essential. Try listening only first and see if the latency is suitable before trying to join in and jam.
```

Author: ```suggestion
Welcome to Jamulus. Please note, latency optimization on Android has not been addressed and experiences will vary strongly from device to device. Trying to participate in jams should only be attempted when you're sure you have a good connection. The usual recommendations apply - a stable connection (4G/5G cellular; Ethernet; WiFi discouraged) and wired, non-Bluetooth headphones are essential. Try listening only first and see if the latency is suitable before trying to join in and jam.
```

Reviewer: Looks good to me.
